### ISSUE TITLE
Non-working using_galpy examples

### ISSUE THREAD

**Issue by michael-petersen** [ilikecubesnstuff/commensurability#3]
In `examples/using_galpy/phase_slice.py`, I currently get two different bugs:

The first is a syntax error:
``` from .interactive import InteractivePhasePlot
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/commensurability/analysis/interactive.py", line 72
    print(self.image[..., *self.indices.values()])
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
SyntaxError: invalid syntax. Perhaps you forgot a comma?
```

This one is surmountable with some tweaking, but I'm guessing it isn't the underlying issue. If I bypass the syntax error and ask for `self.indices.values()`, the current examples return an empty dictionary, which then fails because the example is trying to index the array using non-existent indices. I suspect there is an easy fix?

See same error in `examples/using_galpy/phase.py`

**Comment by ilikecubesnstuff** [comment]
I did some testing - this appears to be a syntax error, not a runtime error that might depend on the contents of `self.indices.values()`. For python 3.8 and older, starred expressions require enclosing parentheses - this is documented in the [Python 3.11 reference](https://docs.python.org/3.11/whatsnew/3.11.html#other-language-changes), but appears to have been changed with [Python 3.9 and newer](https://github.com/python/cpython/issues/90881).

The fix for backwards compatibility was writing explicit parentheses around the starred expression:
```py
    self.im_phase.set_data(self.image[(..., *self.indices.values())].T)
```

For clarity, `self.indices` will be an empty dictionary if the analysis object hosts a 2D image - `self.indices` stores the remaining degrees of freedom not already taken up by the x and y axes in the plot. In the case that is it empty, the expression decomposes into `self.image[...]`, which returns the whole 2D image for plotting.
